<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>YUVPlayer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="mainWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="videoRender" native="true">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="videoSlider">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <widget class="QPushButton" name="btPlay">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>80</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>播放</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btPause">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>50</y>
          <width>80</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>暂停</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btStop">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>50</y>
          <width>80</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
       <widget class="QLabel" name="labTip">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>47</y>
          <width>421</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>7</y>
          <width>55</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>文件路径</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="edFilePath">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>321</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>/home/wangning/HDA/video/yuv420p_320x240.yuv</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btSelectFile">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>10</y>
          <width>80</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>选择文件</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btSetParam">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>10</y>
          <width>80</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>参数设置</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
